### PR TITLE
Add a policy for the file integrity operator

### PR DESCRIPTION
**Pull request by gparvin**
The File integrity operator is a security component that comes with
OpenShift that can help identify runtime changes to nodes that are not
expected.

Signed-off-by: Gus Parvin <gparvin@redhat.com>

### CODE REVIEW

File: community/README.md [hunk 1]
```diff
@@ -127,6 +127,7 @@ Policy  | Description | Prerequisites
...
 [Policy to configure Github identity provider in IDP ](./CM-Configuration-Management/policy-idp-sample-github.yaml) | Use this policy to apply Github OAuth to managed clusters through IDP . | For more information on this operator, see the IDP documentation: [Identity configuration management for Kubernetes](https://identitatem.github.io/idp-mgmt-docs/). NOTE: IDP Operator must be installed before using this policy.
+[Policy to install the OpenShift File Integrity operator](./CM-Configuration-Management/policy-file-integrity-operator.yaml) | Use the file integrity operator to continually run file integrity checks on the cluster nodes. This policy becomes `NonCompliant` when a `FileIntegrityNodeStatus` returns a status of `Failed` which indicates files on the nodes have changed. | OpenShift 4.x is required. See [Understanding the File Integrity Operator](https://docs.openshift.com/container-platform/latest/security/file_integrity_operator/file-integrity-operator-understanding.html) for more details.
```
Comment (dockerymick) [inline community/README.md]: Use the integrity operator file to continually run file integrity checks on the cluster nodes. This policy becomes `NonCompliant` when a `FileIntegrityNodeStatus` returns a status of `Failed`, which indicates files on the nodes have changed.

**Review by dockerymick — COMMENTED** [review]
Made a suggestion

Signed-off-by: Mikela Jackson <mdockery@redhat.com>

File: community/README.md (hunk 1 above)
Comment (gparvin) [reply community/README.md]: @dockerymick Thanks, I made updates based on your comment.  Can you take a look again?

**Review by dockerymick — APPROVED** [review]
Signed-off-by: Mikela Jackson <mdockery@redhat.com>

**Pull request merged**